src/components/EmailPreview/index.jsx

```diff
@@ -0,0 +1,24 @@
+function EmailPreview({emailName, emailSubject, emailBody, emailDate, emailRead}) {
+
+    function backgrCol () {
+        console.log(emailRead)
+        return emailRead==1 ? "bg-white" : "bg-gray-400"
+    }
+
+    return (
+        <div className={backgrCol()+" emailSummaryBox flex justify-between px-4 py-2 border-t-2"}>
+            <div>
+                <p className="text-lg font-bold">{emailName}</p>
+                <p className="text-sm">{emailSubject}</p>
+                <p className="text-sm">{emailBody}</p>
+            </div>
+            
+            <div>
+                <p className="font-bold">{emailDate}</p>
+            </div>
+            
```

Comment:
Remove this empty line - good to have them between opening/closing tags, but not at the end of a chunk like this